docker-compose.yml

```diff
@@ -49,6 +49,8 @@ services:
       - KUZZLE_VAULT_KEY=${KUZZLE_VAULT_KEY}
       - NODE_10_VERSION=10.16.0
       - WITHOUT_KUZZLE=${WITHOUT_KUZZLE}
+      - KUZZLE_SECRETS_FILE=/var/app/features-sdk/fixtures/secrets.enc.json
+      - KUZZLE_VAULT_KEY=secret-password
```

Comment:
Why is there KUZZLE_VAULT_KEY defined twice ?

Reply:
Good catch :sweat_smile: